src/main/java/com/cinema/service/MovieSessionService.java

```diff
@@ -12,4 +13,6 @@
     void update(MovieSession movieSession);
 
     void removeById(Long id);
+
+    Optional<MovieSession> findById(Long movieSessionId);
```

Comment:
```suggestion
    MovieSession get(Long movieSessionId);
```
let's return Optional from DAO layer, and entity from service layer

Reply:
done